data/mods/Aftershock/items/afs_materials.json

```diff
@@ -14,5 +14,23 @@
     "dmg_adj": [ "marked", "dented", "smashed", "shattered" ],
     "bash_dmg_verb": "dented",
     "cut_dmg_verb": "scratched"
+  },
+  {
+    "type": "material",
+    "ident": "graphene_weave",
+    "name": "Graphene Weave",
+    "density": 10,
+    "specific_heat_liquid": 0.82,
```

Comment:
Can we use decimals here?

Reply:
Yes, both specific heat values are floats internally